pkg/webhook/v1alpha2/component/mutating_handler.go

```diff
@@ -125,7 +127,9 @@ func (h *MutatingHandler) Mutate(obj *v1alpha2.Component) error {
 		// copy namespace/label/annotation to the workload and add workloadType label
 		workload.SetNamespace(obj.GetNamespace())
 		workload.SetLabels(util.MergeMap(obj.GetLabels(), map[string]string{WorkloadTypeLabel: workloadType}))
-		workload.SetAnnotations(obj.GetAnnotations())
+		// Add another annotation DefinitionAnnotation which can mark the name of WorkloadDefinition
+		workload.SetAnnotations(util.MergeMap(obj.GetAnnotations(), map[string]string{oam.DefinitionAnnotation: workloadType}))
+		mutatelog.Info("Set annotation definition.oam.dev/name for workload", "annotation value", workloadType)
```

Comment:
this seems to be a debugging log for the testing purpose only

Reply:
Webhook is a little bit for local debug, so I write the log to make sure it takes effect. As you recommended to use the existing label below, I will remove the annotation setting and the log. Thanks.